packages/react-components/react-portal-compat-context/.swcrc

```diff
@@ -2,11 +2,11 @@
   "$schema": "https://json.schemastore.org/swcrc",
   "env": {
     "targets": {
-      "chrome": "84",
-      "edge": "84",
-      "firefox": "75",
-      "opera": "73",
-      "safari": "14.1"
+      "chrome": "79",
```

Comment:
Had to lower the browser support to our partial support matrix here since this package is being used by northstar and is throwing the error below regarding not being able to understand nullish coalescing:

```
verb @fluentui/projects-test test |  [!] (plugin commonjs) SyntaxError: Unexpected token (6:48) in /tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/@fluentui/react-portal-compat-context/lib/PortalCompatContext.js
verb @fluentui/projects-test test |  node_modules/@fluentui/react-portal-compat-context/lib/PortalCompatContext.js (6:48)
verb @fluentui/projects-test test |  4: export const PortalCompatContextProvider = PortalCompatContext.Provider;
verb @fluentui/projects-test test |  5: export function usePortalCompat() {
verb @fluentui/projects-test test |  6:   return React.useContext(PortalCompatContext) ?? portalCompatContextDefaultValue;
verb @fluentui/projects-test test |                                                     ^
verb @fluentui/projects-test test |  7: }
verb @fluentui/projects-test test |  8: //# sourceMappingURL=PortalCompatContext.js.map
verb @fluentui/projects-test test |  SyntaxError: Unexpected token (6:48) in /tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/@fluentui/react-portal-compat-context/lib/PortalCompatContext.js
verb @fluentui/projects-test test |      at Object.pp$4.raise (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:3399:13)
verb @fluentui/projects-test test |      at Object.pp.unexpected (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:1248:8)
verb @fluentui/projects-test test |      at Object.pp$3.parseExprAtom (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2822:10)
verb @fluentui/projects-test test |      at Object.parseExprAtom (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:5836:77)
verb @fluentui/projects-test test |      at Object.pp$3.parseExprSubscripts (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2651:19)
verb @fluentui/projects-test test |      at Object.pp$3.parseMaybeUnary (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2628:17)
verb @fluentui/projects-test test |      at Object.parseMaybeUnary (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:5720:29)
verb @fluentui/projects-test test |      at Object.pp$3.parseExprOps (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2572:19)
verb @fluentui/projects-test test |      at Object.pp$3.parseMaybeConditional (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2555:19)
verb @fluentui/projects-test test |      at Object.pp$3.parseMaybeAssign (/tmp/project-rollup--7012-JvE6lzYMysp0/test-app/node_modules/rollup/dist/shared/rollup.js:2528:19)
verb @fluentui/projects-test test |  

```